Module1.FileSystemVisitor/FileSystemVisitor/FileSystemVisitor.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project
+{
+    public class FileSystemVisitor
+    {
+        private Func<FileSystemInfo, bool> _func;
```

Comment:
rename and set default
```suggestion
        private Func<FileSystemInfo, bool> _fileFilterFun = (x) => true;
```

Reply:
done